subprojects/docs/src/docs/userguide/dependencyManagement.adoc

```diff
@@ -288,13 +288,65 @@ You can also assign collections or arrays of dependency notations to a configura
 
-Since Gradle 1.2 there is also a new programmatic API to access the resolved dependency information. The dependency reports (see the previous paragraph) are using this API under the covers. The API lets you walk the resolved dependency graph and provides information about the dependencies. In future releases the API will grow to provide more information about the resolution result. For more information about the API please refer to the Javadocs on api:org.gradle.api.artifacts.ResolvableDependencies#getResolutionResult[]. Potential usages of the api:org.gradle.api.artifacts.result.ResolutionResult[] API:
+[[sec:listing_dependencies]]
+==== Listing all dependencies in a project
 
-* Creation of advanced dependency reports tailored to your use case.
-* Enabling the build logic to make decisions based on the content of the dependency graph.
+A project can declare one or more dependencies. Gradle can visualize the whole dependency tree for every configuration available in the project with the help of the _dependency report_.
 
+Rendering the dependency tree is particularly useful if you'd like identify which dependencies have been resolved at runtime. It also provides you with information about any dependency conflict resolution that occurred in the process and clearly indicates the selected version. The dependency report always contains declared and transitive dependencies.
+
+Let's say you'd want to create tasks for your project that use the link:https://www.eclipse.org/jgit/[JGit library] to execute SCM operations e.g. to model a release process. You can declare dependencies for any external tooling with the help of a <<sub:scope_of_dependency_configurations,custom configuration>> so that it doesn't doesn't convolute other contexts like the compilation classpath for your production source code. To render the dependencies graph execute the `dependencies` task for the corresponding configuration.
+
+++++
+<sample id="dependencyReport" dir="userguide/dependencies/dependenciesReport" title="Rendering the dependency report for a custom configuration">
+    <sourcefile file="build.gradle" snippet="dependency-declaration" />
+    <output args="-q dependencies --configuration scm"/>
+</sample>
+++++
+
+[NOTE]
+====
+The dependencies report provides detailed information about the dependencies available in the graph. Any dependency that could not be resolved is marked with `FAILED` in red color. Dependencies with the same coordinates that can occur multiple times in the graph are omitted indicated by an asterisk (`*`) character. Dependencies that had to undergo conflict resolution render the requested and selected version separated by a right arrow character.
```

Comment:
This note is a bit lengthy, maybe consider shortening a bit. We probably don't need to spell out "asterisk". 

"that can occur multiple times in the graph are omitted **and** indicated by an asterisk"

Reply:
Maybe we don't even have to make it note. It's just plain information. WDYT? Integrated the other comments from your end.

Reply:
Made this regular text.